tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -104,6 +119,58 @@ async def test_create_sensors(
     )
     assert entity_registry_entry.entity_category == EntityCategory.DIAGNOSTIC
 
+    state = hass.states.get("sensor.heat_meter_heat_usage_mwh")
+    assert not state
+
+
+@patch(API_HEAT_METER_SERVICE)
+async def test_sensors_mwh(mock_heat_meter, hass: HomeAssistant) -> None:
```

Comment:
I think this is a duplicate of the above test, and they could be merged with `pytest.mark.parametrize`